### PR TITLE
[LSAN][NFC] Add a new line to a log

### PR DESCRIPTION
**Pull request by kstoimenov**

### CODE REVIEW

File: compiler-rt/lib/lsan/lsan_common.cpp [hunk 1]
```diff
@@ -762,7 +762,7 @@ static bool CheckForLeaks() {
     VReport(1, "LeakSanitizer is disabled");
```
Comment (hctim) [inline compiler-rt/lib/lsan/lsan_common.cpp:762]: and here?

**Review by hctim — APPROVED** [review]
LGTM, but also maybe do elsewhere in this file?

**Comment by vitalybuka** [comment]
> @llvm/pr-subscribers-clang-modules @llvm/pr-subscribers-backend-aarch64 @llvm/pr-subscribers-clang-static-analyzer-1 @llvm/pr-subscribers-clang @llvm/pr-subscribers-backend-arm @llvm/pr-subscribers-backend-x86 @llvm/pr-subscribers-clang-driver @llvm/pr-subscribers-backend-amdgpu @llvm/pr-subscribers-backend-risc-v
> 
> @llvm/pr-subscribers-compiler-rt-sanitizer
> 
> Changes

That's not nice.

@MaskRay Any idea how to avoid spam like this?
I recently created PR and it had one already merged commit. But I didn't figure out how to remove it from PR.

**Comment by vitalybuka** [comment]
@kstoimenov can you try to `git fetch origin; git rebase origin/main; git push -f <your_fork> HEAD:<fork_branch>`

**Comment by kstoimenov** [comment]
> Why does this have 87 unrelated commits pulled in?

I am learning how to use github. Sorry about the spam.

**Comment by MaskRay** [comment]
> @MaskRay Any idea how to avoid spam like this?
> I recently created PR and it had one already merged commit. But I didn't figure out how to remove it from PR.

After #66320, auto-labeler will not label the patch with more than 10 commits:)

**Pull request closed without merging**